README.md

```diff
@@ -28,89 +30,79 @@ Or, after cloning the repo, go inside the root folder of the project (i.e. along
+Here's how you can interact with a text-based game from within Python using OpenAI's Gym framework.
 
 ```python
-import textworld
-
-env = textworld.start("gen_games/game_1234.ulx")  # Start an existing game.
-agent = textworld.agents.NaiveAgent()  # Or your own `textworld.Agent` subclass.
+import gym
+import textworld.gym
 
-# Collect some statistics: nb_steps, final reward.
-avg_moves, avg_scores = [], []
-N = 10
-for no_episode in range(N):
-    agent.reset(env)  # Tell the agent a new episode is starting.
-    game_state = env.reset()  # Start new episode.
+# Register a text-based game as a new Gym's environment.
+env_id = textworld.gym.register_game("tw_games/custom_game.ulx",
+                                     max_episode_steps=50)
 
-    reward = 0
-    done = False
-    for no_step in range(100):
-        command = agent.act(game_state, reward, done)
-        game_state, reward, done = env.step(command)
+env = gym.make(env_id)  # Start the environment.
 
-        if done:
-            break
+obs, infos = env.reset()  # Start new episode.
+env.render()
 
-    # See https://textworld-docs.maluuba.com/textworld.html#textworld.core.GameState
-    avg_moves.append(game_state.nb_moves)
-    avg_scores.append(game_state.score)
+score, moves, done = 0, 0, False
+while not done:
+    command = input("> ")
+    obs, score, done, infos = env.step(command)
+    env.render()
+    moves += 1
 
 env.close()
-print("avg. steps: {:5.1f}; avg. score: {:4.1f} / 1.".format(sum(avg_moves)/N, sum(avg_scores)/N))
+print("moves: {}; score: {}".format(moves, score))
 ```
 
+> **Note:** To play text-based games without Gym, see [Playing text-based games with TextWorld.ipynb](notebooks/Playing%20text-based%20games%20with%20TextWorld.ipynb)
+
 ## Documentation
+
 For more information about TextWorld, check the [documentation](https://aka.ms/textworld-docs).
 
 ## Notebooks
-Check the [notebooks](notebooks) provided with the framework to see how the framework can be used.
+
+Check the [notebooks](notebooks) provided with the framework to see what you can do with it. You will need the [Jupyter Notebook](https://jupyter.org/install) to run them. You can install it with
+
+    pip install jupyter
 
 ## Citing TextWorld
+
 If you use TextWorld, please cite the following BibTex:
-```
+
+```text
 @Article{cote18textworld,
-  author = {Marc-Alexandre C\^ot\'e and
-            \'Akos K\'ad\'ar and
-            Xingdi Yuan and
-            Ben Kybartas and
-            Tavian Barnes and
-            Emery Fine and
-            James Moore and
-            Matthew Hausknecht and
-            Layla El Asri and
-            Mahmoud Adada and
-            Wendy Tay and
-            Adam Trischler},
+  author = {Marc-Alexandre C\^ot\'e and \'Akos K\'ad\'ar and Xingdi Yuan and Ben Kybartas and Tavian Barnes and Emery Fine and James Moore and Matthew Hausknecht and Layla El Asri and Mahmoud Adada and Wendy Tay and Adam Trischler},
```

Comment:
This was certainly more readable before, did BibTeX not like it?

Reply:
I just tried to save on vertical space (I saw some other repo doing that). I'll revert it back.